static/truewiki/truewiki.css

```diff
@@ -35,8 +35,16 @@ body > nav {
 	height: 32px;
 	margin: 0px auto;
 	min-height: 32px;
-	overflow: hidden;
+	/* If the breadcrumbs are longer than the width of the screen, allow horizontal scrolling */
+	overflow: scroll;
+    white-space: nowrap;
+    scrollbar-width: none;
+}
+
+body > nav::-webkit-scrollbar {
```

Comment:
Meh; is a browser specific exception really needed? And really only webkit? Just checking :)

Reply:
Unfortunately yes. The first one _only_ works for Firefox, and the second, _only_ for Webkit/Blink-based browsers. See the compatibility charts for each:
* https://developer.mozilla.org/en-US/docs/Web/CSS/scrollbar-width
* https://developer.mozilla.org/en-US/docs/Web/CSS/::-webkit-scrollbar